src/measure_theory/function/locally_integrable.lean

```diff
@@ -103,16 +105,24 @@ hg.mul_continuous_on_of_subset hg' hK.measurable_set hK (subset.refl _)
 
 lemma integrable_on.continuous_on_mul_of_subset
   (hg : continuous_on g K) (hg' : integrable_on g' A μ)
-  (hK : is_compact K) (hA : measurable_set A) (hAK : A ⊆ K) :
+  (hA : measurable_set A) (hK : is_compact K) (hAK : A ⊆ K) :
```

Comment:
What's the reason for swapping these arguments?

Reply:
The lemmas `mul_continuous_on_of_subset` and `continuous_on_mul_of_subset` are a left/right pair, identical in all other respects. So it seems strange that the order of arguments is different between the two, and hence I switched the order of `hK` and `hA` in the second one  to match the first.

Reply:
By the same argument, I think you should swap hg and hg'?

Reply:
I think it's good as-is, because in both cases, the functions are listed as hypotheses in the order they appear in the formula.

Reply:
Ah, then I think my view shifts back to my original comment. The order that was here before means that hk and hA appear in the same order as k and A appear in hg and hg'